File: DP/python/9461.py
T = int(input())
dp = [0 for _ in range(101)]
dp[1] = 1
dp[2] = 1
dp[3] = 1
for i in range(4,101):
  dp[i] = dp[i-2] + dp[i-3]
for _ in range(T):
  print(dp[int(input())])

File: DP/javascript/9461.js
const input = require("fs").readFileSync("/dev/stdin").toString().split("\n");

const dp = new Array(101).fill(0);
let result = "";
dp[1] = dp[2] = dp[3] = 1;
for (let i = 4; i < 101; i++) {
  dp[i] = dp[i - 2] + dp[i - 3];
}
for (let i = 1; i < input.length - 1; i++) {
  result += `${dp[input[i]]}\n`;
}
console.log(result);
